doc/doxygen/headers/manifold.h

```diff
@@ -289,14 +289,6 @@
  * (see the documentation of this function). It is also germane to the
  * cases discussed in the @ref GlossDistorted "glossary entry on distorted cells".
  *
- * Another example where the manifold description not just at the boundary but
- * also in the interior of the domain matters is for high-order methods. When
- * using cubic or even higher degrees of the polynomials, full convergence is
- * typically only obtained if a curved description at a boundary transitions
- * into a straight description inside the domain (for example when meshing a
- * ball including the origin) over a layer of finite thickness. This is
- * realized by the class TransfiniteInterpolationManifold.
```

Comment:
Is there equivalent text for this somewhere else? I think that this is an important thing to note.

Reply:
But that is not true, we do not need a finite thickness. We surely need an appropriate placement of points inside the single layer of cells that is adjacent to the boundary with the new `MappingQ` weights, but that should be it. You are right that we should mention the loss of convergence a few paragraphs down (it ends up on this page: http://dealii.org/developer/doxygen/deal.II/group__manifold.html ) but it is really only relevant if the points are not placed carefully. I will push a second commit with this extension.

Reply:
You're right.